djangocms_versioning/models.py

```diff
@@ -174,6 +179,8 @@ def can_be_archived(self):
         return can_proceed(self._set_archive)
 
     def archive(self, user):
+        # trigger pre operation signal
+        action_token = send_pre_version_operation(constants.OPERATION_ARCHIVE, version=self)
```

Comment:
Why are you putting this ahead of the docstring? The docstring should be the first thing in a function, surely?